src/Avalonia.Controls.ColorPicker/ColorView/ColorViewTab.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Avalonia.Controls
+{
+    /// <summary>
+    /// Defines a specific subview/tab within the <see cref="ColorView"/>.
+    /// </summary>
+    /// <remarks>
+    /// This is indexed to match the default control template ordering.
+    /// </remarks>
+    public enum ColorViewTab
```

Comment:
Terminology for "Tab" in this enum and in the SelectedTabIndex should be mentioned/discussed. I'm attempting to make this control template-agnostic so it can be fully re-templated however one would like. However, these two areas need a word and I chose "Tab". Those that re-template this control may not have tabs at all but would still like some form of "subviews" (group boxes, etc.). So the term here may not be good.

While the term "Tab" is used in the API, in comments "subview/tab" is used which is a bit more generic. All terms I can think of are "Tabs", "Panels", "Subviews".

Does anyone have comments or concerns here?

Reply:
I want to just throw in "Page" here. It's really hard to find the right word, so I have no preference here.

Reply:
Either Tab or Page - we use both terms in the project. TabControl, TabStrip, but IPageTransition.
I don't think we use Subviews anywhere, except from UIKit/AppKit related code (terminology used by Apple).

Reply:
Subview feels like the correct term here even if it isn't used much in .NET or Avalonia. All the other terms are overly specific and won't necessary match with how the control is re-templated in the future. So in comments I'm using "tab/panel/page (subview)" but in the API "Tab" is removed as mentioned above everywhere but the enum.